### PR TITLE
Update README with contributor info

### PR DESCRIPTION
**Pull request by bobbypage**
Updating readme with some details about the core team / contributors.

PTAL @iwankgb @Creatone @dims @mrunalp @haircommander @dashpole 

Thanks for all your contributions & help on cAdvisor!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,20 @@ cAdvisor aims to improve the resource usage and performance characteristics of r
 ## Community
 
 Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Slack](https://kubernetes.slack.com) in the #sig-node channel (get an invitation [here](http://slack.kubernetes.io/)). We also have [discuss.kubernetes.io](https://discuss.kubernetes.io/).
+
+Please reach out and get involved in the project, we're actively looking for more contributers to bring on board!
```
Comment (Creatone) [inline README.md:71]: nit: `s/contributers/contributors/`

File: README.md [hunk 2]
```diff
@@ -67,3 +67,20 @@ cAdvisor aims to improve the resource usage and performance characteristics of r
...
+
+### Core Team
+* [@bobbypage, Google](http://github.com/bobbypage)
+* [@iwankgb, Independent](http://github.com/iwankgb)
+* [@creatone, Intel](http://github.com/creatone)
+* [@dims, VMWare](http://github.com/dims)
+* [@mrunalp, RedHat](http://github.com/mrunalp)
+
+### Frequent Collaborators
+* [@pehunt, RedHat](http://github.com/pehunt)
```
Comment (haircommander) [inline README.md:81]: nit, my username is haircommander :slightly_smiling_face:

File: README.md (hunk 1 above)
Comment (bobbypage) [reply README.md:71]: Done

File: README.md (hunk 2 above)
Comment (bobbypage) [reply README.md]: oops, fixed thanks!

**Review by iwankgb — APPROVED** [review]
So much fame ❤️

**Pull request merged**